src/components/mc/cpu/mc_cpu.c

```diff
@@ -70,11 +70,11 @@ static ucc_status_t ucc_mc_cpu_mem_free(void *ptr)
     return UCC_OK;
 }
 
-static ucc_status_t ucc_mc_cpu_mem_type(const void *ptr,
-                                        ucc_memory_type_t *mem_type)
+static ucc_status_t ucc_mc_cpu_mem_query(const void *ptr, size_t length,
+                                        ucc_mem_attr_t *mem_attr)
 {
     if (ptr == NULL) {
-        *mem_type = UCC_MEMORY_TYPE_HOST;
+        mem_attr->mem_type = UCC_MEMORY_TYPE_HOST;
```

Comment:
pls set length to zero to make linter happy
```
mc_cpu.c:73:66: warning: parameter 'length' is unused [misc-unused-parameters]
static ucc_status_t ucc_mc_cpu_mem_query(const void *ptr, size_t length,
```

Reply:
where is this log show linter warnings? can we set Jenkins check fails when this happens so that we can look in it?

Reply:
why length is needed at all?

Reply:
not needed here. but when querying base alloc length and MC do not support it, it will set what we pass to the query.

Reply:
Linter results are here https://github.com/openucx/ucc/pull/72/checks?check_run_id=1882307244
We agreed to not block PR merge even if linter complains about something because there was a lot of warnings initially. Now all old warnings are fixed and we can change linter behavior.

Reply:
> where is this log show linter warnings? can we set Jenkins check fails when this happens so that we can look in it?

I have set merge to block, if CI complains. It seems to be in clear now.